### ISSUE TITLE
Dfsu.contains() fails in some cases for global meshes

### ISSUE THREAD

**Issue by jsmariegaard** [DHI/mikeio#280]
@caichac-dhi reported an issue for dfsu file made from a global mesh which is due to a problem with the determination of the boundary polyline for a global mesh.

**Comment by ecomodeller** [comment]
Somehow in everyday life it is easy to forget that we live on a sphere. 🌎

**Comment by ecomodeller** [comment]
Related to issue #697 , i.e. not only a problem with global meshes

**Comment by daniel-caichac-DHI** [comment]
It is kind of addresses with this open PR right https://github.com/DHI/mikeio/pull/700 ?

**Comment by ecomodeller** [comment]
> It is kind of addressed with this open PR right #700 ?

Yes, but I would like some feedback on it before merging it, to avoid having to revert or to fix it again.
